openvdb/openvdb/tree/RootNode.h

```diff
@@ -1057,6 +1084,9 @@ inline
 RootNode<ChildT>::RootNode(const RootNode<OtherChildType>& other,
     const ValueType& backgd, const ValueType& foregd, TopologyCopy)
     : mBackground(backgd)
+#if OPENVDB_ABI_VERSION_NUMBER >= 10
+    , mOrigin(0, 0, 0)
```

Comment:
Should there be some kind of check to confirm that other.origin() == Coord(0, 0, 0)?

In a hypothetical world where VDB 10.0 only supports zero origins and 10.1 adds support for non-zero origins, it should be possible for a VDB with a non-zero origin to be reinterpret casted from 10.1 to 10.0 and this copy-constructor run that quietly adjusts the origin that could be unexpected for the user? Throwing here would at least inform them that they need to re-root _before_ trying to copy-construct from this grid.